common/app/views/fragments/immersiveGalleryHeadTonal.scala.html

```diff
@@ -0,0 +1,90 @@
+@(item: model.ContentType, showBadge: Boolean = false, showMeta: Boolean = true)(implicit request: RequestHeader)
+
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<header class="content__head content__head--immersive content__head--desktop is-fixed-height tonal__head tonal__head--@toneClass(item)">
```

Comment:
Thanks for your comments @johnduffell and @sammorrisdesign.

The only really common part between the two headers is the style tag. I have put that new template in the common folder named very verbosely `immersiveHeaderBackgroundStyling`. It is in common as it is shared between articles and applications - this can change when we have the separate module that @johnduffell may create.

`headerImmersive` remains in article, and `galleryHeader` is in applications as that is where they are used. 

Hopefully that makes things more clear. Let me know if this is better!